src/main/java/dev/dokan/dokan_java/UnsignedNumbers.java

```diff
@@ -0,0 +1,22 @@
+package dev.dokan.dokan_java;
+
+
+public class UnsignedNumbers {
+
+    public static String toUnsignedString(@Unsigned byte value) {
+        return toUnsignedString(Byte.toUnsignedInt(value));
+    }
+
+    public static String toUnsignedString(@Unsigned short value) {
+        return toUnsignedString(Short.toUnsignedInt(value));
+    }
+
+    public static String toUnsignedString(@Unsigned int value) {
+        return Integer.toUnsignedString(value);
+    }
+
+    public static String toUnsignedString(@Unsigned long value) {
+        return Long.toUnsignedString(value);
+    }
+
```

Comment:
As discussed: It would be nice to have also delegate methods for unsigned division and other operations, which are different from their signed counterparts.

Reply:
Resolved by commits 8a15e47 and 236347d